### ISSUE TITLE
[Mac OSX] Atom crashes since update to 1.9.4

### ISSUE THREAD

**Issue by lukasoppermann** [atom/atom#12335]
When open Atom, or drag a folder onto atom or open a folder via the file menu (somehow I can open atom and my projects via the cli) I get the following:

![screen shot 2016-08-05 at 11 37 23](https://cloud.githubusercontent.com/assets/813754/17432748/15baa668-5b01-11e6-923e-fdcae763b929.png)

**Comment by lukasoppermann** [comment]
This has not changed, even though I am on 1.9.6 now.

Deleting the `Application Support/Atom` folder did not help either. What else can I try?

**Comment by lukasoppermann** [comment]
That works, just like opening folders via cli (as mentioned above).

**Comment by lukasoppermann** [comment]
@mnquintana any ideas?

**Comment by 50Wliu** [comment]
@lukasoppermann Can you paste the output of `atom --version` here?  Just want to make sure of something.

**Comment by lukasoppermann** [comment]
Sure:

```
Atom    : 1.9.6
Electron: 0.37.8
Chrome  : 49.0.2623.75
Node    : 5.10.0
```

**Comment by 50Wliu** [comment]
Does `~/.atom` exist, and if so, does `~/.atom/config.cson` (or `.json`) exist as well?

**Comment by lukasoppermann** [comment]
Hey `~/.atom` and `~/.atom/config.cson` exist. (However `~/.atom/config.json` does NOT)

**Comment by 50Wliu** [comment]
This may be related to #12196.

**Comment by snodnipper** [comment]
Atom    : 1.9.8
Electron: 0.37.8
Chrome  : 49.0.2623.75
Node    : 5.10.0

<img width="518" alt="atom_crashed" src="https://cloud.githubusercontent.com/assets/576719/17588629/3a965dbc-5fc8-11e6-9c0b-3cc22ce60189.png">

**Comment by lukasoppermann** [comment]
This is different I think, mine was solved with the solution for #12196

**Comment by snodnipper** [comment]
Opening from the terminal with "atom ." seemed to resolve this.

The above screen was caused (repeatably) by double clicking atom from the Applications folder...with all atom files deleted.
